README.md

```diff
@@ -217,26 +217,31 @@ As an alternative, one can also specify remote builder as usual in
 `/etc/nix/machines` or via the `nix.buildMachines` nixos options in
 `configuration.nix`. This allows to parallelize builds across multiple machines.
 
-## GitHub api token
+## GitHub API token
 
 Some commands (i.e., `post-result` or `merge`) require a GitHub API token, and
 even for read-only calls, GitHub returns 403 error messages if your IP hits the
 rate limit for unauthenticated calls.
 
-To use a token, first create a
-[personal access token](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/).
-If you plan to post comments with the reports generated, you need to add the
-`repo:public_repo` scope.
+Nixpkgs-review will automatically read the oauth_token stored by [hub][1] or
+[gh][2] if they are installed.
+
+Otherwise, you'll have to manually create a "personal access token (classic)"
+through the GitHub website. See [the GitHub documentation][3] for
```

Comment:
```suggestion
Otherwise, you'll have to create a "personal access token (classic)"
through GitHub's website. See [the GitHub documentation](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#personal-access-tokens-classic) for
```